specification/network/resource-manager/readme.python.md

```diff
@@ -18,6 +18,7 @@ Generate all API versions currently shipped for this package
 
 ```yaml $(python) && $(multiapi)
 batch:
+  - tag: package-2019-12
```

Comment:
Is this PR change the same content of https://github.com/Azure/azure-rest-api-specs/pull/8731 this one

Reply:
yes. the commit for network merged into previous PR #8731